lib/bambora/v1/payment_resource.rb

```diff
@@ -3,15 +3,24 @@
 module Bambora
   module V1
     class PaymentResource
-      # Summary: Create and modify payments.
-      # Note: The link below links to all apis includding profiles and tokenization. There aren't great docs explaining
-      #       the /payments endpoints alone.
-      # Docs: https://dev.na.bambora.com/docs/references/payment_APIs/
-      #       https://dev.na.bambora.com/docs/references/payment_SDKs/take_payments/?shell#
-      # Endpoint: https://api.na.bambora.com/v1/payments
-      def initialize(client:, sub_path:)
+      def initialize(client:)
         @client = client
-        @sub_path = sub_path
+        @sub_path = '/v1/payments'
+      end
+
+      def make_payment(payment_data)
+        @client.request(method: :post, path: @sub_path, body: payment_data)
+      end
+
+      def make_payment_with_payment_profile(customer_code:, amount:)
+        make_payment(
+          amount: amount,
+          payment_method: 'payment_profile',
+          payment_profile: {
+            customer_code: customer_code,
+            card_id: 1,
```

Comment:
Is this always going to `1`? What does `1` mean?

Reply:
This is a parameter that we have included in our own codebase. Within a Bambora payment profile, it is possible to store multiple cards. Specifying the card ID allows you to use one of multiple cards on the same profile. Since we do not currently use this functionality, this parameter should always be set to `1`.

I realize this looks like a magic number, so I have extracted it to a class-level constant.

Reply:
Since this is a public gem though, shouldn't we allow users to pass `card_id` into the method?